frontend/app/filters/index.js

```diff
@@ -32,6 +32,8 @@ export function durationHumanize(duration) {
 export function scheduleHumanize(schedule) {
   if (schedule === null) {
     return 'Never';
+  } else if (schedule.split(' ').length === 5) {
+    return `CRON tab: ${schedule}`;
```

Comment:
We could but I am not a fan at all of pop-over because they work very poorly with touch devices. But if you really want it, I can implement it.